src/components/NavBar/AccountMenu.tsx

```diff
@@ -71,15 +87,18 @@ const AccountMenu = () => {
         anchorEl={anchorEl}
         onClose={handleClose}
         anchorOrigin={{
-          vertical: "top",
+          vertical: -5,
           horizontal: "center",
         }}
         transformOrigin={{
           vertical: "bottom",
           horizontal: "center",
         }}
-        sx={{ ".MuiPopover-paper": styles.popover }}
+        sx={{
```

Comment:
i just realized you could also use `PaperProps` and then `sx` within that, if you think that looks better feel free to make that change